<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>450</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::Computer"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="MainName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>170</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Калькулятор&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="resultLayout">
     <property name="spacing">
      <number>60</number>
     </property>
     <item>
      <widget class="QLabel" name="resultNumber">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Результат&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resultText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="inputLayout" stretch="0,0">
     <property name="spacing">
      <number>35</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="firstNumLayout">
       <property name="spacing">
        <number>60</number>
       </property>
       <item>
        <widget class="QLabel" name="firstNumber">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;1-е Число&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="firstNumberEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="secondNumLayout">
       <property name="spacing">
        <number>60</number>
       </property>
       <item>
        <widget class="QLabel" name="secondNumber">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;2-е Число&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondNumberEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>344</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ButtonsAll">
     <item>
      <layout class="QVBoxLayout" name="operLayouts">
       <item>
        <layout class="QHBoxLayout" name="operLayout_1">
         <item>
          <widget class="QPushButton" name="sumButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Сложение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="subButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Вычитание</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mulButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Умножение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="divButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Деление</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="operLayout_2">
         <item>
          <widget class="QPushButton" name="powButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Степень</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="percentButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Процент</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="clcButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Очистка</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
